README.md

```diff
@@ -62,6 +62,17 @@ CPMAddPackage("uri@version#tag")
 
 In the shorthand syntax if the URI is of the form `gh:user/name`, it is interpreted as GitHub URI and converted to `https://github.com/user/name.git`. If the URI is of the form `gl:user/name`, it is interpreted as a [GitLab](https://gitlab.com/explore/) URI and coverted to `https://gitlab.com/user/name.git`. Otherwise the URI used verbatim as a git URL. All packages added using the shorthand syntax will be added using the [EXCLUDE_FROM_ALL](https://cmake.org/cmake/help/latest/prop_tgt/EXCLUDE_FROM_ALL.html) flag.
 
+The single-argument syntax also works for URLs:
+
+```cmake
+# An archive package from a given url. The version is inferred
+CPMAddPackage("https://example.com/my-package-1.2.3.zip")
+# An archive package from a given url with an MD5 hash provided
+CPMAddPackage("https://example.com/my-package-1.2.3.zip#68e20f674a48be38d60e129f600faf7d")
```

Comment:
Doesn't CMake use `URL_HASH`s in the form [`URL_HASH <algo>=<hashValue>`](https://cmake.org/cmake/help/latest/module/ExternalProject.html)? So should the syntax rather be like

```cmake
CPMAddPackage(
  "https://example.com/my-package-1.2.3.zip#MD5=68e20f674a48be38d60e129f600faf7d"
)
```

Or is MD5 the default algorithm? I think we should probably document and test either case.

Reply:
MD5 is the default. But I'll document that others are possible

Reply:
Could we also get a test case for `<algo>=<hash>`? Just so future changes in the regex won't break the feature.

Reply:
Yes. Will do in a couple of hours.
BTW even though I clearly remember reading that MD5 is the default, I can't find this anywhere. I opened a [issue on CMake's tracker](https://gitlab.kitware.com/cmake/cmake/-/issues/21859) to make sure